tools/ansible/tasks/logging/test/logging_test.go

```diff
@@ -37,7 +55,9 @@ func TestPluginsInstalled(t *testing.T) {
 		installedVersion, ok := installedPlugins[pluginName]
 		assert.Truef(t, ok, "Missing plugin %s", pluginName)
 		if pluginVersion != "" {
-			assert.Equalf(t, pluginVersion, installedVersion,
+			// Getting only the version in the format 1.5 instead of 1.5.6, because installation automatically
+			// fetches the latest subversion
+			assert.Equalf(t, pluginVersion, installedVersion[:3],
```

Comment:
Not all versions will be semver so this wont work

Reply:
Correct. I'll mae the assert here only specific to Fluentd and call it out in the comments.